attestation-service/attestation-service/src/lib.rs

```diff
@@ -115,10 +130,9 @@ impl AttestationService {
 
     /// Set Attestation Verification Policy.
     pub async fn set_policy(&mut self, input: SetPolicyInput) -> Result<()> {
```

Comment:
this is still anyhow, should/can we convert it?

Reply:
This one has dependency from `kbs/src/api/src/attestation/coco/builtin.rs:25`.  we can ignore this one as part of initial refactoring. Later on things can be extended to broader directories one all the basic refactoring is done in these modules. wdyt ?

Reply:
yup